src/types.db

```diff
@@ -235,6 +235,7 @@ swap_io                 value:DERIVE:0:U
 tcp_connections         value:GAUGE:0:4294967295
 temperature             value:GAUGE:U:U
 threads                 value:GAUGE:0:U
+thermal_throttle        core_throttle_count:COUNTER:0:U, package_throttle_count:COUNTER:0:U
```

Comment:
We require new types to have a single *data source* only and that that *data source* be named "value". Also, `COUNTER` must only be used for 32bit counters that overflow frequently (more than once per hour). All other cumulative metrics must use `DERIVE`.

I recommend to create a new type like this:

```
total_throttles value:DERIVE:0:U
```

Then dispatch your metrics as

*   thermal_throttle-cpu0/total_throttles-core
*   thermal_throttle-cpu0/total_throttles-package
*   …
*   thermal_throttle-cpu*n*/total_throttles-core
*   thermal_throttle-cpu*n*/total_throttles-package

In other words, set `vl.plugin_instance` to `"cpu%d"` and `vl.type_instance` to either `"core"` or `"package"`.